### PR TITLE
Added Cook Class

### PR DESCRIPTION
**Pull request by Azzzzzy**
Added [Cook Class](https://www.dmsguild.com/product/232212/The-Cook-Class) from DMs Guild

### CODE REVIEW

File: third-party/dms-guild.index [hunk 1]
```diff
@@ -15,6 +15,10 @@
 		<!-- feats -->
 		<file name="expanded-racial-feats.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/third-party/dms-guild/expanded-racial-feats.xml" />		
 		
+		<!-- classes -->
+		<file name="class-cook.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/third-party/dms-guild/class-cook.xml" />
```
Comment (swdriessen) [inline third-party/dms-guild.index]: when updating existing files, please update the version number at the top of the file

File: third-party/dms-guild/class-cook.xml [hunk 2]
```diff
@@ -0,0 +1,1379 @@
...
+    <description>
+      <p>At 6th level, you modify your Cook's Bag to use the vitality coursing through a recently taken piece of flesh to cook it into a morsel.</p>
+      <p>When you deal slashing or piercing damage to a hostile creature that isn't a humanoid, celestial, undead, construct, or fey, with a melee weapon attack, you can cut off a chunk of meat and place it into your Cook's Bag as part of the same attack. The chunk is immediately seasoned and cooked into the following morsel by your Cook's Bag. You can't have more than one morsel of this type in your bag at a time.</p>
+      <p>Your Cook's bag can season and cook the following morsel a number of times equal to your Constitution modifier (minimum of once). You regain expended uses when you finish a long rest.</p>
+    </description>
+    <sheet>
+      <description>When you deal piercing or slashing damage you can cook 1 Fresh Cut Morsel as a bonus action. You can hold only 1 Fresh Cut and can cook Fresh Cut {{constitution:modifier}} times per long rest.</description>
+    </sheet>
+    <rules>
+      <grant type="Class Feature" name="ID_DMSG_COOK_CLASS_FEATURE_COOK_MORSEL_FRESH_CUT" />
```
Comment (ColdAnkles) [inline third-party/dms-guild/class-cook.xml]: This grant throws an error for me as it can't find this.

Comment (Azzzzzy) [reply third-party/dms-guild/class-cook.xml]: Fixed it thanks for catching it!

**Comment by swdriessen** [comment]
Cleanup, fixed some things, change id abbreviation from **COOK** to **TCC** (see commits for more details)

In case the questions comes up again, it still slows down the UI (class tab) at higher levels (depending on the machine it might be significant as was reported back then) but i'll be using this class to test and improve that overtime. It's just something people will have to live with for now if they want to use the class at higher levels.

**Pull request merged**